packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

```diff
@@ -128,12 +133,14 @@ export const AddNewTrustline: FC = () => {
     const id = Number(urlParams.get('id')) || 0;
     const memos = parseMemos(urlParams.get('memos'));
     const flags = parseTrustSetFlags(urlParams.get('flags'));
+    const inAppCall = urlParams.get('inAppCall') === 'true' || false;
```

Comment:
This is already declared on line 55, I think we can remove it here.

Reply:
Done